pyproject.toml

```diff
@@ -1,59 +1,63 @@
-[tool.poetry]
-package-mode = false
+[project]
 name = "romm"
-version = "0.0.1"
 description = "A beautiful, powerful, self-hosted rom manager"
 license = "GNU AGPLv3"
-repository = "https://github.com/rommapp/romm"
-authors = ["Zurdi <zurdi@romm.app>", "Arcane <arcane@romm.app>"]
+readme = "README.md"
+authors = [
+  { name = "Zurdi", email = "zurdi@romm.app" },
+  { name = "Arcane", email = "arcane@romm.app" },
+]
```

Comment:
missing your name and email here lol

Reply:
You're right! I finally added myself haha